modules/openapi-generator/src/main/resources/cpp-qt5-client/HttpFileElement.cpp.mustache

```diff
@@ -57,7 +57,7 @@ QJsonValue {{prefix}}HttpFileElement::asJsonValue() const {
     if (!result) {
         qDebug() << "Error opening file " << local_filename;
     }
-    return QJsonDocument::fromBinaryData(bArray.data()).object();
+    return QJsonDocument::fromJson(bArray.data()).object();
```

Comment:
Is this backward compatible?
`fromBinaryData` is deprecated only in 5.15

Reply:
Tested in 5.9 LTS

Reply:
Added another version check since build failed.